### PR TITLE
docs(observability): update docs after consolidating metrics, logs, traces into observability

### PR DESCRIPTION
**Pull request by slonka**
Signed-off-by: slonka <slonka@users.noreply.github.com>

After this PR: https://github.com/kumahq/kuma/pull/4308/files we need to update docs accordingly. I updated:

- the namespaces to [mesh-observability](https://github.com/kumahq/kuma/pull/4308/files#diff-4097df65e73d6b3a2813df0ea3c87f15a6c43ba3aca9b73b3e60346758ef0344R152)
- urls pre/post fixes to match the new namespace

Related: 
- https://github.com/kumahq/kuma-website/issues/815
- https://github.com/kumahq/kuma-website/pull/830

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits) yes

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)? yes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,23 @@ Once this is done you can go in `explore` and pick the kuma-datasource with the
...
 Add to the datasource configuration:
 
+#### kuma >= 1.7.0
+
+```yaml
+    datasources:
+      - name: Prometheus
+        type: prometheus
+        access: proxy
+        url: http://prometheus-server.mesh-observability
```
Comment (slonka) [inline README.md:46]: this line is changed

File: README.md [hunk 2]
```diff
@@ -51,7 +68,13 @@ Add to the datasource configuration:
 
 ### With `kumactl`
 
-If you use `kumactl install metrics` with a version of kumactl >= 1.3.0 the plugin will be setup automatically.
+#### kumactl >= 1.7.0
+
+If you use `kumactl install observability --components grafana,prometheus` with a version of kumactl >= 1.7.0 the plugin will be setup automatically.
```
Comment (lahabana) [inline README.md:73]: Don't specify the components. It's just better if jaeger is there as well.

Comment (slonka) [reply README.md:73]: so this applies probably to everything, right?

Comment (lahabana) [reply README.md]: yes

**Review by lahabana — APPROVED** [review]

**Pull request merged**